nanome/_internal/_structure/_serialization/_workspace_serializer.py

```diff
@@ -20,15 +20,15 @@ def name(self):
     def serialize(self, version, value, context):
         context.write_using_serializer(self.array, value._complexes)
 
-        context.write_using_serializer(self.vector, value._position)
-        context.write_using_serializer(self.quaternion, value._rotation)
-        context.write_using_serializer(self.vector, value._scale)
+        context.write_using_serializer(self.pos, value._position)
+        context.write_using_serializer(self.rot, value._rotation)
+        context.write_using_serializer(self.pos, value._scale)
```

Comment:
Is this intentionally using a `UnityPositionSerializer`?

Reply:
nooope